packages/salesforcedx-vscode-core/package.json

```diff
@@ -172,6 +174,10 @@
           "command": "sfdx.force.function.start",
           "when": "sfdx:project_opened && resource =~ /.*/functions/.*/[^/]+(/[^/]+\\.(ts|js|java|json|toml))?$/ && resourceFilename != package.json && resourceFilename != package-lock.json && resourceFilename != tsconfig.json"
         },
+        {
+          "command": "sfdx.force.function.container.start",
+          "when": "sfdx:project_opened && resource =~ /.*/functions/.*/[^/]+(/[^/]+\\.(ts|js|java|json|toml))?$/ && resourceFilename != package.json && resourceFilename != package-lock.json && resourceFilename != tsconfig.json"
+        },
```

Comment:
Note the previous `sfdx.force.function.start` was left unaltered here, but is updated to run a new command function in index.ts.